redbot/cogs/audio/core/commands/player.py

```diff
@@ -37,7 +37,11 @@ class PlayerCommands(MixinMeta, metaclass=CompositeMetaClass):
     @commands.guild_only()
     @commands.bot_has_permissions(embed_links=True)
     async def command_play(self, ctx: commands.Context, *, query: str):
-        """Play a URL or search for a track."""
+        """Play a URL or search for a track.
+
+        To play a local track/file, the query should be <parentfolder>/<filename>.
```

Comment:
Given that the known majority of users are on windows, please use windows paths here.